bin/commands/generate.mjs

```diff
@@ -125,22 +123,29 @@ export default {
   async action(opts) {
     const docs = await loadAndParse(opts.input, opts.ignore);
 
-    if (!opts.skipLint && !runLint(docs)) {
-      console.error('Lint failed; aborting generation.');
-      process.exit(1);
-    }
+    // if (!opts.skipLint && !runLint(docs)) {
+    //   console.error('Lint failed; aborting generation.');
+    //   process.exit(1);
+    // }
 
-    const { runGenerators } = createGenerator(docs);
     const { getAllMajors } = createNodeReleases(opts.changelog);
 
-    await runGenerators({
-      generators: opts.target,
-      input: opts.input,
-      output: opts.output && resolve(opts.output),
-      version: coerce(opts.version),
-      releases: await getAllMajors(),
-      gitRef: opts.gitRef,
-      threads: parseInt(opts.threads, 10),
-    });
+    const releases = await getAllMajors();
+
+    await Promise.all(
+      docs.map(async doc => {
+        const { runGenerators } = createGenerator(doc);
+
+        await runGenerators({
+          generators: opts.target,
+          input: opts.input,
+          output: opts.output && resolve(opts.output),
+          version: coerce(opts.version),
+          releases,
+          gitRef: opts.gitRef,
+          threads: parseInt(opts.threads, 10),
+        });
+      })
+    );
```

Comment:
I know this PR is still a draft, but I'm concerned that this will overload the system's CPU. We already run each generator on it's own thread, why do we need to iterate on each doc as well?

Reply:
@avivkeller It was a misunderstanding, fixed now. Can you take a look?